src/pages/game/lib/GameBackground.tsx

```diff
@@ -0,0 +1,31 @@
+import type { NextPage } from 'next';
+import Image from 'next/image';
+import scene4 from '@/public/images/gameBG/scene4.svg';
+import scene7 from '@/public/images/gameBG/scene17.svg';
+export default function GameBackground(bg: any) {
```

Comment:
```suggestion
export default function GameBackground({ bg }: { bg: string }) {
```

Try avoiding any type.